org.chromium.Chromium.yaml

```diff
@@ -157,7 +157,7 @@ modules:
     sources:
       - type: git
         url: https://github.com/refi64/flextop
-        commit: 485c332c7ec36a8b74be8f3622fc756ac6263f40
```

Comment:
Nice!

I can't work out how to take a screenshot of Win+Tab but I can work out how to take a screenshot of the Endless OS taskbar.

![Capture d’écran de 2021-04-07 21-47-44](https://user-images.githubusercontent.com/86760/113931871-e5e40700-97ea-11eb-9661-7a7e80ed5af8.png)